src/components/Truncate/Truncate.tsx

```diff
@@ -117,14 +120,37 @@ export class Truncate extends React.PureComponent<
     )
 
     const shouldShowTooltip = showTooltipOnTruncate && this.state.isTruncated
-    const word = this.getTruncatedContent()
```

Comment:
Changing this variable name from `word` to `truncatedText` since it may not always be a word that needs truncating and, in the case of email addresses where I'll be using this feature, I think `text` makes more sense. 

Naming it `truncatedText` is a little bit of a misnomer because there are cases when the text will come back unchanged - for example, if the limit is 15 characters and the text is less than that - but I think it helps show that the text is probably being manipulated.